src/navigations/HomeBottomTabs.js

```diff
@@ -35,16 +35,9 @@ function HomeBottomTabs() {
   const navigation = useNavigation();
   const {isLoadingFetchingChannelDetail} = usePushNotificationHook();
   const {updateAppBadgeWith} = useAppBadgeHook();
-  const isMounted = React.useRef(true);
 
   React.useEffect(() => {
-    return () => {
-      isMounted.current = false;
-    };
-  }, []);
-
-  React.useEffect(() => {
-    if (otherProfileData !== null && initialStartup.id !== null && isMounted.current) {
+    if (otherProfileData !== null && initialStartup.id !== null) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `isMounted` ref and its associated cleanup effect is a good change. The `isMounted` ref was used to prevent state updates on unmounted components, but in this case, it's unnecessary because navigation actions are safe to call even after a component has unmounted. Therefore, the code looks cleaner and more straightforward without it.
```diff
- if (otherProfileData !== null && initialStartup.id !== null && isMounted.current) {
+ if (otherProfileData !== null && initialStartup.id !== null) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->